### ISSUE TITLE
missing markRecord for Order

### ISSUE THREAD

**Issue by lukyrys** [riesenia/pohoda#15]
```
The option "markRecord" does not exist. Defined options are: "activity", "carrier", "centre", "contract", "date", "dateDelivery", "dateFrom", "dateTo", "intNote", "isExecuted", "isReserved", "myIdentity", "note", "number", "numberOrder", "orderType", "partnerIdentity", "paymentType", "priceLevel", "regVATinEU", "text".
```
```
<xsd:element name="markRecord" type="typ:boolean" minOccurs="0" default="true">
<xsd:annotation>
<xsd:documentation>
Označení záznamu v programu POHODA, sloupec "X". Výchozí hodnota je "označený záznam".
</xsd:documentation>
</xsd:annotation>
</xsd:element>
```
https://www.stormware.cz/schema/version_2/order.xsd

**Comment by segy** [comment]
feel free to submit a pr

**Comment by lukyrys** [comment]
@segy PR #16 is it ok?

**Comment by segy** [comment]
fixed in v1.4.12
